lib/cvss.js

```diff
@@ -36,6 +36,27 @@ function CVSS(vector) {
     return roundUp(Math.min(ISC + exploitability - 0.1, 10), 1);
   }
 
+  /**
+   * Parses the vector to a rating
+   *
+   * @returns {String} Rating value
+   */
+  function getRating() {
+    const score = getScore();
+    let rating = "None";
+
+    if (score <= 3.9) {
```

Comment:
I think you need to check if the score is `> 0` here, because with a score of `<= 0` it returns `Low` instead of `None`

Reply:
sorry about that! Tested all but 0 :smile: Fixed now!
0 -> None
0.1 -> Low

and fixed the linting parts!